tests/ui/pages/desktop/collections.py

```diff
@@ -0,0 +1,31 @@
+from pypom import Region
+from selenium.webdriver.common.by import By
+
+from base import Base
+
+
+class Collections(Base):
+    """ Collections page. """
+
+    _root_locator = (By.CLASS_NAME, 'island')
+
+    _name_locator = (By.CSS_SELECTOR, ".item > .info > h3")
+
+    def wait_for_page_to_load(self):
+        self.wait.until(lambda _: self.find_element(
+            *self._name_locator)).is_displayed()
```

Comment:
So this will wait for the first collection name to be displayed. You can instead do this as `self.collections[0].is_displayed` and implement an `is_displayed` property for `Collection`.